src/nix_file.rs

```diff
@@ -190,9 +190,7 @@ impl NixFile {
             // Something like `foo`, `"foo"` or `${"foo"}`
```

Comment:
While testing, I found out that this is the branch also used for the `foo` in `inherit (bar) foo` in Nix > 2.20.

```suggestion
            // Something like `foo`, `"foo"` or `${"foo"}`
            // For Nix versions > 2.20, this is also `foo` in `inherit (bar) foo`
```